### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Boomer-1**
## Description

updating the readme to show the current version cap of 1.20.4

## Proposed changes

simply changing 1.20.x to read 1.20.4 until we update to the latest version

## Checklist

- [x ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [ ] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.16.* - 1.20.*).
- [ x] I followed the existing code standards and didn't mess up the formatting.
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Here is a full summary of the differences between the two different versions of
 
 | | development (latest) | "stable" |
 | ------------------ | -------- | -------- |
-| **Minecraft version(s)** | :video_game: **1.16.\* - 1.20.\*** | :video_game: **1.16.\* - 1.20.\*** |
+| **Minecraft version(s)** | :video_game: **1.16.\* - 1.20.4\*** | :video_game: **1.16.\* - 1.20.4\*** |
```
Comment (ybw0014) [inline README.md:33]:
```suggestion
| **Minecraft version(s)** | :video_game: **1.16.\* - 1.20.4** | :video_game: **1.16.\* - 1.20.4** |
```

the `\*` is used to display a *.

**Review by ybw0014 — CHANGES_REQUESTED** [review]

**Review by J3fftw1 — DISMISSED** [review]

**Review by JustAHuman-xD — APPROVED** [review]

**Review by Sfiguz7 — APPROVED** [review]
It's been hanging for so long that at this point we might as well correct it I guess

**Comment by ybw0014** [comment]
need to change a line of code to make CI check pass

File: README.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ Here is a full summary of the differences between the two different versions of
 
 | | development (latest) | "stable" |
 | ------------------ | -------- | -------- |
-| **Minecraft version(s)** | :video_game: **1.16.\* - 1.20.\*** | :video_game: **1.16.\* - 1.20.\*** |
+| **Minecraft version(s)** | :video_game: **1.16.\* - 1.20.4** | :video_game: **1.16.\* - 1.20.4** |
```
Comment (Sfiguz7) [inline README.md:33]:
```suggestion
| **Minecraft version(s)** | :video_game: **1.16.\* - 1.20.4** | :video_game: **1.16.\* - 1.20.4** | 
```

**Comment by Sfiguz7** [comment]
Wait, you mean a line of the source code, right?

**Comment by ybw0014** [comment]
yeah a line of the source code

**Comment by ybw0014** [comment]
This can be closed when #4186 is merged.

**Comment by ybw0014** [comment]
this can be closed

**Comment by WalshyDev** [comment]
Ez

**Pull request closed without merging**